cypress/integration/event-signup-flow.spec.js

```diff
@@ -747,4 +747,26 @@ describe("Verify page load " + new Date(), () => {
 			});
 		});
 	});
+	it('It shows the "Past Online Events" page', function () {
+		cy.navigateToPage("/events/category/online-events/archive");
+		cy.verifyPageHeading("Past Online Events");
+		searchForEvent
+			.getSearchforEventsHeading()
+			.should("exist")
+			.should("have.text", "Search for Past Online Events");
+	});
+	it("It shows past events only", function () {
+		cy.navigateToPage("/events/category/online-events/archive");
+		var today = new Date();
+		var currentDate = today.getDate() + " " + today.getMonth() + " " + today.getFullYear();
+		cy.getPastEventDateAndTime().each((eventDateAndTime, index, $list) => {
+			eventDateAndTime = eventDateAndTime.text().split(":")[0];
+			var eventDate = eventDateAndTime.substring(0, eventDateAndTime.lastIndexOf(" "));
+			if (eventDate < currentDate) {
```

Comment:
Can `getPastEventDateAndTime` parse the date text into a `Date` object and then we can simplify this quite a bit to be:

```
expect(eventDate.getTime()).toBeLessThan(new Date().getTime())
```

Reply:
Fixed.